spec/namespaces/cat.yaml

```diff
@@ -2388,47 +2404,47 @@ components:
     cat.segments::query.bytes:
       in: query
       name: bytes
-      description: The unit used to display byte values.
+      description: The units used to display byte values.
       schema:
         $ref: '../schemas/_common.yaml#/components/schemas/ByteUnit'
       style: form
     cat.segments::query.cluster_manager_timeout:
       name: cluster_manager_timeout
       in: query
-      description: Operation timeout for connection to cluster-manager node.
+      description: A timeout for connection to the cluster manager node.
```

Comment:
"The" timeout? Or, even better, do we mean "The amount of time allowed to establish a connection to the cluster manager node"?